src/DotLiquid/Block.cs

```diff
@@ -13,11 +13,8 @@ namespace DotLiquid
     /// </summary>
     public class Block : Tag
     {
-        private static readonly Regex IsTag = R.B(@"^{0}", Liquid.TagStart);
-        private static readonly Regex IsVariable = R.B(@"^{0}", Liquid.VariableStart);
-        private static readonly Regex ContentOfVariable = R.B(@"^{0}(.*){1}$", Liquid.VariableStart, Liquid.VariableEnd);
 
-        internal static readonly Regex FullToken = R.B(@"^{0}\s*(\w+)\s*(.*)?{1}$", Liquid.TagStart, Liquid.TagEnd);
+        internal static readonly Regex FullToken = R.B(@"^{0}\s*(\w+)\s*((?s:.*)?){1}$", Liquid.TagStart, Liquid.TagEnd);
```

Comment:
Oh boy, which regex syntax is that? The {0} and {1} are those C# string interpolations or actual regex pieces?

Reply:
yep, it's combining regex for start, finish, and anything in between into one. Not my idea...